flow/Knobs.cpp

```diff
@@ -284,6 +284,19 @@ void FlowKnobs::initialize(Randomize randomize, IsSimulated isSimulated) {
 	init( RESTCLIENT_MAX_CONNECTION_LIFE,                      120 );
 	init( RESTCLIENT_REQUEST_TRIES,                             10 );
 	init( RESTCLIENT_REQUEST_TIMEOUT_SEC,                      120 );
+
+	// HTTP KMS Connector
+	init( REST_KMS_CONNECTOR_KMS_DISCOVERY_URLS,                 "");
+	// Configuration mode to enable HttpKmsConnector to procure validation tokens.
```

Comment:
I agree, these should all be server knobs

Reply:
Will move them to server knobs. Thanks.